README.md

```diff
@@ -247,23 +256,35 @@ cmake --build . --parallel 8
 
 We have created a docker image that best simulates the old travis system (currently it's ubuntu xenial 16.04). To build nomacs in a docker, you have to create the image:
 
-````bash
+```bash
 docker build --rm -f "Dockerfile" -t nomacs:latest empty-docker-dir
-`````
+```
 
 To deploy nomacs in a docker on your system, you can mount this directory using:
 
-````bash
+```bash
 docker run --rm -it -v C:\\coding\\nomacs:/usr/nomacs nomacs:latest
-````
+```
 
 If needed, you can upload the image:
 
-````bash
+```bash
 docker login
 docker tag nomacs diemmarkus/nomacs
 docker push diemmarkus/nomacs:latest
-````
+```
+
+
+## Enable Debug output
```

Comment:
This part regarding Debug output is no longer valid.

Reply:
Ok, you are right. I just looked in the code and if I understand that right debug messages are not created at all!

https://github.com/nomacs/nomacs/blob/84bfa5e6617d3fc133c921687f8f6fed96007c2f/ImageLounge/src/DkCore/DkUtils.cpp#L438

The Qt default behavior activates qDebug() output by not defining `QT_NO_DEBUG_OUTPUT` and this is done by `CMAKE_BUILD_TYPE=Debug`. But now I don't get qDebug() output even with this define. Have I overseen something?

Reply:
I would suggest changing 
https://github.com/nomacs/nomacs/blob/84bfa5e6617d3fc133c921687f8f6fed96007c2f/ImageLounge/src/DkCore/DkUtils.cpp#L438
to
```
    QString p = "%{if-info}[INFO] %{endif}%{if-warning}[WARNING] %{endif}%{if-critical}[CRITICAL] %{endif}%{if-fatal}[ERROR] %{endif}";
#ifndef QT_NO_DEBUG_OUTPUT
    p += "%{if-debug}[DEBUG] %{endif}";
#endif
    p += "%{message}";

```

Reply:
If you like I can do a new PR to not mix it up here. Just let me know.

Reply:
close this and make a new PR.

Reply:
Ok, I have undone this change and will open a new PR addressing the debug issue. Would be nice to merge this now. Tx.